src/FoldersTree.m

```diff
@@ -529,7 +518,8 @@ -(NSInteger)nextFolderWithUnreadAfterNode:(TreeNode *)startingNode
 
 		// If we've gone full circle and not found
 		// anything, we're out of unread articles
-		if (nextNode == startingNode)
+		if (nextNode == startingNode
```

Comment:
Done, but regarding code's legibility, I'd prefer investing in text comments :-)